src/js/dqa-scores.jsx

```diff
@@ -0,0 +1,357 @@
+        var firstSiteData = props.data[Object.keys(props.data)[0]];
+        var statusList = Object.keys(firstSiteData);
+        var w = props.columnWidth;
+
+        var columnWidths = {
+            site: w,
+            score: w,
+        };
+        for (var status in statusList) {
+            columnWidths[status] = w;
+        }
+
+        var rows = this._initRows(props);
+
+        return {
+            columnWidths: columnWidths,
+            rows: rows,
+            sortBy: 'score',
+            sortDir: SortTypes.DESC,
+            statusList: statusList,
+        };
+    },
+
+    componentWillReceiveProps: function(nextProps) {
+        this.setState(this.getInitContents(nextProps));
+    },
+
+    componentWillMount: function() {
+        this.setState(this.getInitContents(this.props));
+    },
+
+    componentDidMount: function() {
+        // set a timeout for updating width, otherwise everything on the page gets
+        // re-mounted on any navigation. I don't quite understand why, but somehow 
+        // this is caused by the fact that _updateWidth accesses dimentions of an element.
+        setTimeout(this._updateWidth);
+
+        if (window.addEventListener) {
+          window.addEventListener('resize', this._onResize, false);
+        } else if (window.attachEvent) {
+          window.attachEvent('onresize', this._onResize);
+        } else {
+          window.onresize = this._onResize;
+        }
+    },
+
+    _onResize: function() {
+        this._updateWidth();
+    },
+
+    _updateWidth: function() {
+        // there has to be a better way to size the table correctly, but this will do for now.
+        // (TODO figure out a cleaner way to do this)
+        var component = document.getElementById('dqaContainer');
+        if (component) {
+            this.setState({
+                tableWidth : component.clientWidth
+            });
+        }
+    },
+
+    _onColumnResizeEndCallback: function(newColumnWidth, dataKey) {
+        w = this.state.columnWidths
+        w[dataKey] = newColumnWidth;
+        this.setState({
+            columnWidths: w
+        })
+        isColumnResizing = false;
+    },
+
+    _renderHeader: function(label, cellDataKey) {
+        var tooltip_text = '';
+        if (Status[cellDataKey]) {
+            tooltip_text = Status[cellDataKey].desc;
+        }
+
+        return (
+          <a onClick={this._sortRowsBy.bind(null, cellDataKey)} data-toggle='tooltip' title={tooltip_text}>{label}</a>
+        );
+    },
+
+    _rowGetter: function (rowIndex) {
+        return this.state.rows[rowIndex];
+    },
+
+    render: function() {
+        var colWidth = this.props.columnWidth;
+        var rowHeight = 30;
+        var headerHeight = 50;
+        var numRows = this.state.rows.length || 0;
+        
+        var sortDirArrow = '';
+    
+        if (this.state.sortDir !== null){
+          sortDirArrow = this.state.sortDir === SortTypes.ASC ? ' ↓' : ' ↑';
```

Comment:
The arrows are a bit thin in the grid, see what the [unicode triangles](http://stackoverflow.com/a/5853994/407954) look like.